src/renderer/contexts/GlobalNodeState.tsx

```diff
@@ -927,16 +927,10 @@ export const GlobalProvider = memo(
         }, [changeNodes, changeEdges]);
         const selectAllFn = useCallback(() => {
             changeNodes((nodes) => {
-                return nodes.map((n) => ({
-                    ...n,
-                    selected: true,
-                }));
+                return nodes.map((n) => ({ ...n, selected: true }));
```

Comment:
Could also just get rid of the brackets and the return

Reply:
Done.